conversation/message_test.go

```diff
@@ -31,27 +31,61 @@ func TestCreateMessage(t *testing.T) {
 }
 
 func TestListMessages(t *testing.T) {
-	mbtest.WillReturnTestdata(t, "messageListObject.json", http.StatusOK)
-	client := mbtest.Client(t)
+	t.Run("limit_offset", func(t *testing.T) {
+		mbtest.WillReturnTestdata(t, "messageListObject.json", http.StatusOK)
+		client := mbtest.Client(t)
 
-	messageList, err := ListMessages(client, "convid", DefaultListOptions)
-	if err != nil {
-		t.Fatalf("unexpected error listing Messages: %s", err)
-	}
+		messageList, err := ListMessages(client, "convid", DefaultListOptions)
```

Comment:
As `nil` and `DefaultOptions` resolve to the same thing, perhaps you can pass some other value for `ListOptions` here? That way we assert that it's actually being used